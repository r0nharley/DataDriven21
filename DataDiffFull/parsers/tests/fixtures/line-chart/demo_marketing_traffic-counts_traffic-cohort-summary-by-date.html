<svg version="1.1" class="highcharts-root" style="font-family:&quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, Arial, Helvetica, sans-serif;font-size:12px;" xmlns="http://www.w3.org/2000/svg" width="1998" height="467">
    <desc>Created with Highstock 5.0.6</desc>
    <defs>
        <clipPath id="highcharts-tashkhr-148">
            <rect x="0" y="0" width="1930" height="383" fill="none"></rect>
        </clipPath>
        <clipPath id="highcharts-tashkhr-149">
            <rect x="0" y="0" width="1930" height="383" fill="none"></rect>
        </clipPath>
    </defs>
    <rect fill="#ffffff" class="highcharts-background" x="0" y="0" width="1998" height="467" rx="0" ry="0"></rect>
    <rect fill="none" class="highcharts-plot-background" x="58" y="10" width="1930" height="383"></rect>
    <g class="highcharts-grid highcharts-xaxis-grid ">
        <path fill="none" class="highcharts-grid-line" d="M 232.5 10 L 232.5 393" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 408.5 10 L 408.5 393" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 583.5 10 L 583.5 393" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 759.5 10 L 759.5 393" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 934.5 10 L 934.5 393" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 1110.5 10 L 1110.5 393" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 1285.5 10 L 1285.5 393" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 1461.5 10 L 1461.5 393" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 1636.5 10 L 1636.5 393" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 1812.5 10 L 1812.5 393" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 1987.5 10 L 1987.5 393" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 57.5 10 L 57.5 393" opacity="1"></path>
    </g>
    <g class="highcharts-grid highcharts-yaxis-grid ">
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 58 393.5 L 1988 393.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 58 329.5 L 1988 329.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 58 265.5 L 1988 265.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 58 201.5 L 1988 201.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 58 138.5 L 1988 138.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 58 74.5 L 1988 74.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 58 10.5 L 1988 10.5" opacity="1"></path>
    </g>
    <rect fill="none" class="highcharts-plot-border" x="58" y="10" width="1930" height="383"></rect>
    <g class="highcharts-axis highcharts-xaxis ">
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 232.5 393 L 232.5 403" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 408.5 393 L 408.5 403" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 583.5 393 L 583.5 403" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 759.5 393 L 759.5 403" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 934.5 393 L 934.5 403" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 1110.5 393 L 1110.5 403" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 1285.5 393 L 1285.5 403" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 1461.5 393 L 1461.5 403" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 1636.5 393 L 1636.5 403" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 1812.5 393 L 1812.5 403" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 1987.5 393 L 1987.5 403" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 57.5 393 L 57.5 403" opacity="1"></path>
        <path fill="none" class="highcharts-axis-line" stroke="#ccd6eb" stroke-width="1" d="M 58 393.5 L 1988 393.5"></path>
    </g>
    <g class="highcharts-axis highcharts-yaxis ">
        <path fill="none" class="highcharts-axis-line" d="M 58 10 L 58 393"></path>
    </g>
    <g class="highcharts-series-group ">
        <g class="highcharts-series highcharts-series-0 highcharts-spline-series highcharts-color-undefined " transform="translate(58,10) scale(1 1)" width="383" height="1930" clip-path="url(https://secure.rentlytics.com/demo/bi/marketing-traffic-counts/visualization#highcharts-tashkhr-149)">
            <path fill="none" d="M 87.727272727273 55.34349999999995 C 87.727272727273 55.34349999999995 193.0000000000012 38.619166666666615 263.18181818182 38.619166666666615 C 333.363636363636 38.619166666666615 368.454545454544 78.37456666666706 438.63636363636 92.94133333333332 C 508.81818181818 107.5081000000004 543.9090909090901 96.61813333333289 614.09090909091 111.45299999999997 C 684.272727272726 126.2878666666662 719.363636363634 141.76106666666735 789.54545454545 167.11566666666664 C 859.7272727272699 192.47026666666736 894.8181818181799 238.226 965 238.226 C 1035.1818181818 238.226 1070.2727272727 143.10156666665745 1140.4545454545 110.62316666666663 C 1210.6363636363399 78.14476666665743 1245.72727272726 95.94150000000569 1315.9090909091 75.83399999999995 C 1386.0909090909001 55.72650000000567 1421.1818181818003 10.085666666666611 1491.3636363636 10.085666666666611 C 1561.5454545454402 10.085666666666611 1596.6363636363599 111.44023333331202 1666.8181818182 186.0103333333333 C 1737 260.58043333331204 1842.2727272727 382.9361666666667 1842.2727272727 382.9361666666667" class="highcharts-graph" stroke="#fbb755" stroke-width="2" stroke-linejoin="round" stroke-linecap="round"></path>
            <path fill="none" d="M 77.727272727273 55.34349999999995 L 87.727272727273 55.34349999999995 C 87.727272727273 55.34349999999995 193.0000000000012 38.619166666666615 263.18181818182 38.619166666666615 C 333.363636363636 38.619166666666615 368.454545454544 78.37456666666706 438.63636363636 92.94133333333332 C 508.81818181818 107.5081000000004 543.9090909090901 96.61813333333289 614.09090909091 111.45299999999997 C 684.272727272726 126.2878666666662 719.363636363634 141.76106666666735 789.54545454545 167.11566666666664 C 859.7272727272699 192.47026666666736 894.8181818181799 238.226 965 238.226 C 1035.1818181818 238.226 1070.2727272727 143.10156666665745 1140.4545454545 110.62316666666663 C 1210.6363636363399 78.14476666665743 1245.72727272726 95.94150000000569 1315.9090909091 75.83399999999995 C 1386.0909090909001 55.72650000000567 1421.1818181818003 10.085666666666611 1491.3636363636 10.085666666666611 C 1561.5454545454402 10.085666666666611 1596.6363636363599 111.44023333331202 1666.8181818182 186.0103333333333 C 1737 260.58043333331204 1842.2727272727 382.9361666666667 1842.2727272727 382.9361666666667 L 1852.2727272727 382.9361666666667" stroke-linejoin="round" visibility="visible" stroke="rgba(192,192,192,0.0001)" stroke-width="22" class="highcharts-tracker"></path>
        </g>
        <g class="highcharts-markers highcharts-series-0 highcharts-spline-series highcharts-color-undefined highcharts-tracker " transform="translate(58,10) scale(1 1)" width="383" height="1930" clip-path="none">
            <path fill="#fbb755" d="M 438 92.94133333333332 C 438 92.94133333333332 438 92.94133333333332 438 92.94133333333332 C 438 92.94133333333332 438 92.94133333333332 438 92.94133333333332 Z" class="highcharts-halo highcharts-color-undefined" fill-opacity="0.25"></path>
            <path fill="#fbb755" d="M 438 86.94133333333332 C 445.992 86.94133333333332 445.992 98.94133333333332 438 98.94133333333332 C 430.008 98.94133333333332 430.008 86.94133333333332 438 86.94133333333332 Z" stroke="#ffffff" stroke-width="1" visibility="hidden"></path>
        </g>
        <g class="highcharts-series highcharts-series-1 highcharts-spline-series highcharts-color-undefined " transform="translate(58,10) scale(1 1)" width="383" height="1930" clip-path="url(https://secure.rentlytics.com/demo/bi/marketing-traffic-counts/visualization#highcharts-tashkhr-149)">
            <path fill="none" d="M 87.727272727273 283.1008333333333 C 87.727272727273 283.1008333333333 193.0000000000012 285.0796666666666 263.18181818182 288.9735 C 333.363636363636 292.8673333333333 368.454545454544 298.88043333333343 438.63636363636 302.57 C 508.81818181818 306.25956666666673 543.9090909090901 303.2849333333332 614.09090909091 307.42133333333334 C 684.272727272726 311.5577333333332 719.363636363634 315.3239000000002 789.54545454545 323.252 C 859.7272727272699 331.1801000000002 894.8181818181799 347.0618333333333 965 347.0618333333333 C 1035.1818181818 347.0618333333333 1070.2727272727 327.5671333333313 1140.4545454545 320.4433333333333 C 1210.6363636363399 313.31953333333126 1245.72727272726 315.4771000000012 1315.9090909091 311.44283333333334 C 1386.0909090909001 307.4085666666678 1421.1818181818003 300.272 1491.3636363636 300.272 C 1561.5454545454402 300.272 1666.8181818182 342.9765 1666.8181818182 342.9765" class="highcharts-graph" stroke="#fc7570" stroke-width="2" stroke-linejoin="round" stroke-linecap="round"></path>
            <path fill="none" d="M 77.727272727273 283.1008333333333 L 87.727272727273 283.1008333333333 C 87.727272727273 283.1008333333333 193.0000000000012 285.0796666666666 263.18181818182 288.9735 C 333.363636363636 292.8673333333333 368.454545454544 298.88043333333343 438.63636363636 302.57 C 508.81818181818 306.25956666666673 543.9090909090901 303.2849333333332 614.09090909091 307.42133333333334 C 684.272727272726 311.5577333333332 719.363636363634 315.3239000000002 789.54545454545 323.252 C 859.7272727272699 331.1801000000002 894.8181818181799 347.0618333333333 965 347.0618333333333 C 1035.1818181818 347.0618333333333 1070.2727272727 327.5671333333313 1140.4545454545 320.4433333333333 C 1210.6363636363399 313.31953333333126 1245.72727272726 315.4771000000012 1315.9090909091 311.44283333333334 C 1386.0909090909001 307.4085666666678 1421.1818181818003 300.272 1491.3636363636 300.272 C 1561.5454545454402 300.272 1666.8181818182 342.9765 1666.8181818182 342.9765 L 1676.8181818182 342.9765" stroke-linejoin="round" visibility="visible" stroke="rgba(192,192,192,0.0001)" stroke-width="22" class="highcharts-tracker"></path>
        </g>
        <g class="highcharts-markers highcharts-series-1 highcharts-spline-series highcharts-color-undefined highcharts-tracker " transform="translate(58,10) scale(1 1)" width="383" height="1930" clip-path="none">
            <path fill="#fc7570" d="M 614 307.42133333333334 C 614 307.42133333333334 614 307.42133333333334 614 307.42133333333334 C 614 307.42133333333334 614 307.42133333333334 614 307.42133333333334 Z" class="highcharts-halo highcharts-color-undefined" fill-opacity="0.25"></path>
            <path fill="#fc7570" d="M 614 301.42133333333334 C 621.992 301.42133333333334 621.992 313.42133333333334 614 313.42133333333334 C 606.008 313.42133333333334 606.008 301.42133333333334 614 301.42133333333334 Z" stroke="#ffffff" stroke-width="1" visibility="hidden"></path>
        </g>
        <g class="highcharts-series highcharts-series-2 highcharts-spline-series highcharts-color-undefined " transform="translate(58,10) scale(1 1)" width="383" height="1930" clip-path="url(https://secure.rentlytics.com/demo/bi/marketing-traffic-counts/visualization#highcharts-tashkhr-149)">
            <path fill="none" d="M 87.727272727273 338.6358333333333 C 87.727272727273 338.6358333333333 193.0000000000012 339.9891 263.18181818182 340.87 C 333.363636363636 341.7509 368.454545454544 342.03176666666667 438.63636363636 343.0403333333333 C 508.81818181818 344.0489 543.9090909090901 343.7041999999999 614.09090909091 345.9128333333333 C 684.272727272726 348.12146666666655 719.363636363634 349.94710000000015 789.54545454545 354.0835 C 859.7272727272699 358.2199000000001 894.8181818181799 366.5948333333333 965 366.5948333333333 C 1035.1818181818 366.5948333333333 1070.2727272727 360.40299999999894 1140.4545454545 356.8283333333333 C 1210.6363636363399 353.25366666666565 1245.72727272726 351.84933333333424 1315.9090909091 348.7215 C 1386.0909090909001 345.59366666666756 1421.1818181818003 341.18916666666667 1491.3636363636 341.18916666666667 C 1561.5454545454402 341.18916666666667 1596.6363636363599 356.904933333331 1666.8181818182 365.2543333333333 C 1737 373.6037333333309 1842.2727272727 382.9361666666667 1842.2727272727 382.9361666666667" class="highcharts-graph" stroke="#6EDA55" stroke-width="2" stroke-linejoin="round" stroke-linecap="round"></path>
            <path fill="none" d="M 77.727272727273 338.6358333333333 L 87.727272727273 338.6358333333333 C 87.727272727273 338.6358333333333 193.0000000000012 339.9891 263.18181818182 340.87 C 333.363636363636 341.7509 368.454545454544 342.03176666666667 438.63636363636 343.0403333333333 C 508.81818181818 344.0489 543.9090909090901 343.7041999999999 614.09090909091 345.9128333333333 C 684.272727272726 348.12146666666655 719.363636363634 349.94710000000015 789.54545454545 354.0835 C 859.7272727272699 358.2199000000001 894.8181818181799 366.5948333333333 965 366.5948333333333 C 1035.1818181818 366.5948333333333 1070.2727272727 360.40299999999894 1140.4545454545 356.8283333333333 C 1210.6363636363399 353.25366666666565 1245.72727272726 351.84933333333424 1315.9090909091 348.7215 C 1386.0909090909001 345.59366666666756 1421.1818181818003 341.18916666666667 1491.3636363636 341.18916666666667 C 1561.5454545454402 341.18916666666667 1596.6363636363599 356.904933333331 1666.8181818182 365.2543333333333 C 1737 373.6037333333309 1842.2727272727 382.9361666666667 1842.2727272727 382.9361666666667 L 1852.2727272727 382.9361666666667" stroke-linejoin="round" visibility="visible" stroke="rgba(192,192,192,0.0001)" stroke-width="22" class="highcharts-tracker"></path>
        </g>
        <g class="highcharts-markers highcharts-series-2 highcharts-spline-series highcharts-color-undefined highcharts-tracker " transform="translate(58,10) scale(1 1)" width="383" height="1930" clip-path="none">
            <path fill="#6EDA55" d="M 965 366.5948333333333 C 965 366.5948333333333 965 366.5948333333333 965 366.5948333333333 C 965 366.5948333333333 965 366.5948333333333 965 366.5948333333333 Z" class="highcharts-halo highcharts-color-undefined" fill-opacity="0.25"></path>
            <path fill="#6EDA55" d="M 965 361 C 972.992 361 972.992 373 965 373 C 957.008 373 957.008 361 965 361 Z" stroke="#ffffff" stroke-width="1" visibility="hidden"></path>
        </g>
        <g class="highcharts-series highcharts-series-3 highcharts-spline-series highcharts-color-undefined " transform="translate(58,10) scale(1 1)" width="383" height="1930" clip-path="url(https://secure.rentlytics.com/demo/bi/marketing-traffic-counts/visualization#highcharts-tashkhr-149)">
            <path fill="none" d="M 87.727272727273 351.0195 C 87.727272727273 351.0195 193.0000000000012 352.0791333333333 263.18181818182 352.5515 C 333.363636363636 353.0238666666666 368.454545454544 352.79406666666665 438.63636363636 353.3813333333333 C 508.81818181818 353.9686 543.9090909090901 353.7260333333333 614.09090909091 355.48783333333336 C 684.272727272726 357.2496333333333 719.363636363634 359.19016666666676 789.54545454545 362.19033333333334 C 859.7272727272699 365.1905000000001 894.8181818181799 370.4886666666667 965 370.4886666666667 C 1035.1818181818 370.4886666666667 1070.2727272727 365.17773333333264 1140.4545454545 362.76483333333334 C 1210.6363636363399 360.3519333333327 1245.72727272726 360.39023333333387 1315.9090909091 358.4241666666667 C 1386.0909090909001 356.45810000000057 1421.1818181818003 352.9345 1491.3636363636 352.9345 C 1561.5454545454402 352.9345 1596.6363636363599 366.85016666666496 1666.8181818182 372.8505 C 1737 378.8508333333316 1842.2727272727 382.9361666666667 1842.2727272727 382.9361666666667" class="highcharts-graph" stroke="#218a8c" stroke-width="2" stroke-linejoin="round" stroke-linecap="round"></path>
            <path fill="none" d="M 77.727272727273 351.0195 L 87.727272727273 351.0195 C 87.727272727273 351.0195 193.0000000000012 352.0791333333333 263.18181818182 352.5515 C 333.363636363636 353.0238666666666 368.454545454544 352.79406666666665 438.63636363636 353.3813333333333 C 508.81818181818 353.9686 543.9090909090901 353.7260333333333 614.09090909091 355.48783333333336 C 684.272727272726 357.2496333333333 719.363636363634 359.19016666666676 789.54545454545 362.19033333333334 C 859.7272727272699 365.1905000000001 894.8181818181799 370.4886666666667 965 370.4886666666667 C 1035.1818181818 370.4886666666667 1070.2727272727 365.17773333333264 1140.4545454545 362.76483333333334 C 1210.6363636363399 360.3519333333327 1245.72727272726 360.39023333333387 1315.9090909091 358.4241666666667 C 1386.0909090909001 356.45810000000057 1421.1818181818003 352.9345 1491.3636363636 352.9345 C 1561.5454545454402 352.9345 1596.6363636363599 366.85016666666496 1666.8181818182 372.8505 C 1737 378.8508333333316 1842.2727272727 382.9361666666667 1842.2727272727 382.9361666666667 L 1852.2727272727 382.9361666666667" stroke-linejoin="round" visibility="visible" stroke="rgba(192,192,192,0.0001)" stroke-width="22" class="highcharts-tracker"></path>
        </g>
        <g class="highcharts-markers highcharts-series-3 highcharts-spline-series highcharts-color-undefined highcharts-tracker " transform="translate(58,10) scale(1 1)" width="383" height="1930" clip-path="none"></g>
        <g class="highcharts-series highcharts-series-4 highcharts-spline-series highcharts-color-undefined " transform="translate(58,10) scale(1 1)" width="383" height="1930" clip-path="url(https://secure.rentlytics.com/demo/bi/marketing-traffic-counts/visualization#highcharts-tashkhr-149)">
            <path fill="none" d="M 87.727272727273 355.1048333333333 C 87.727272727273 355.1048333333333 193.0000000000012 357.4028333333333 263.18181818182 357.4028333333333 C 333.363636363636 357.4028333333333 368.454545454544 356.89216666666664 438.63636363636 356.89216666666664 C 508.81818181818 356.89216666666664 543.9090909090901 357.2368666666666 614.09090909091 358.871 C 684.272727272726 360.5051333333333 719.363636363634 362.50950000000006 789.54545454545 365.06283333333334 C 859.7272727272699 367.61616666666674 894.8181818181799 371.6376666666667 965 371.6376666666667 C 1035.1818181818 371.6376666666667 1070.2727272727 366.5437666666661 1140.4545454545 364.616 C 1210.6363636363399 362.68823333333273 1245.72727272726 361.9988333333333 1315.9090909091 361.9988333333333 C 1386.0909090909001 361.9988333333333 1421.1818181818003 361.9988333333333 1491.3636363636 361.9988333333333 C 1561.5454545454402 361.9988333333333 1666.8181818182 379.553 1666.8181818182 379.553" class="highcharts-graph" stroke="#9b9bd7" stroke-width="2" stroke-linejoin="round" stroke-linecap="round"></path>
            <path fill="none" d="M 77.727272727273 355.1048333333333 L 87.727272727273 355.1048333333333 C 87.727272727273 355.1048333333333 193.0000000000012 357.4028333333333 263.18181818182 357.4028333333333 C 333.363636363636 357.4028333333333 368.454545454544 356.89216666666664 438.63636363636 356.89216666666664 C 508.81818181818 356.89216666666664 543.9090909090901 357.2368666666666 614.09090909091 358.871 C 684.272727272726 360.5051333333333 719.363636363634 362.50950000000006 789.54545454545 365.06283333333334 C 859.7272727272699 367.61616666666674 894.8181818181799 371.6376666666667 965 371.6376666666667 C 1035.1818181818 371.6376666666667 1070.2727272727 366.5437666666661 1140.4545454545 364.616 C 1210.6363636363399 362.68823333333273 1245.72727272726 361.9988333333333 1315.9090909091 361.9988333333333 C 1386.0909090909001 361.9988333333333 1421.1818181818003 361.9988333333333 1491.3636363636 361.9988333333333 C 1561.5454545454402 361.9988333333333 1666.8181818182 379.553 1666.8181818182 379.553 L 1676.8181818182 379.553" stroke-linejoin="round" visibility="visible" stroke="rgba(192,192,192,0.0001)" stroke-width="22" class="highcharts-tracker"></path>
        </g>
        <g class="highcharts-markers highcharts-series-4 highcharts-spline-series highcharts-color-undefined highcharts-tracker " transform="translate(58,10) scale(1 1)" width="383" height="1930" clip-path="none"></g>
    </g>
    <g class="highcharts-legend" transform="translate(710,426)">
        <rect fill="none" class="highcharts-legend-box" rx="0" ry="0" x="0" y="0" width="578" height="26" visibility="visible"></rect>
        <g>
            <g>
                <g class="highcharts-legend-item highcharts-spline-series highcharts-color-undefined highcharts-series-0" transform="translate(8,3)">
                    <path fill="none" d="M 0 11 L 16 11" class="highcharts-graph" stroke="#fbb755" stroke-width="2"></path>
                    <text x="21" style="color:#333333;font-size:12px;font-weight:normal;cursor:pointer;fill:#333333;" text-anchor="start" y="15">
                        <tspan>First Contacts</tspan>
                    </text>
                </g>
                <g class="highcharts-legend-item highcharts-spline-series highcharts-color-undefined highcharts-series-1 " transform="translate(135.6875,3)">
                    <path fill="none" d="M 0 11 L 16 11" class="highcharts-graph" stroke="#fc7570" stroke-width="2"></path>
                    <text x="21" y="15" style="color:#333333;font-size:12px;font-weight:normal;cursor:pointer;fill:#333333;" text-anchor="start">Shows</text>
                </g>
                <g class="highcharts-legend-item highcharts-spline-series highcharts-color-undefined highcharts-series-2 " transform="translate(215.3125,3)">
                    <path fill="none" d="M 0 11 L 16 11" class="highcharts-graph" stroke="#6EDA55" stroke-width="2"></path>
                    <text x="21" y="15" style="color:#333333;font-size:12px;font-weight:normal;cursor:pointer;fill:#333333;" text-anchor="start">
                        <tspan>Apps Submitted</tspan>
                    </text>
                </g>
                <g class="highcharts-legend-item highcharts-spline-series highcharts-color-undefined highcharts-series-3 " transform="translate(355.453125,3)">
                    <path fill="none" d="M 0 11 L 16 11" class="highcharts-graph" stroke="#218a8c" stroke-width="2"></path>
                    <text x="21" y="15" style="color:#333333;font-size:12px;font-weight:normal;cursor:pointer;fill:#333333;" text-anchor="start">
                        <tspan>Apps Approved</tspan>
                    </text>
                </g>
                <g class="highcharts-legend-item highcharts-spline-series highcharts-color-undefined highcharts-series-4 " transform="translate(491.859375,3)">
                    <path fill="none" d="M 0 11 L 16 11" class="highcharts-graph" stroke="#9b9bd7" stroke-width="2"></path>
                    <text x="21" y="15" style="color:#333333;font-size:12px;font-weight:normal;cursor:pointer;fill:#333333;" text-anchor="start">Move-Ins</text>
                </g>
            </g>
        </g>
    </g>
    <g class="highcharts-axis-labels highcharts-xaxis-labels ">
        <text x="145.72727272727275" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:165px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="412" opacity="1">
            <tspan>Jul 2016</tspan>
        </text>
        <text x="321.1818181818182" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:165px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="412" opacity="1">
            <tspan>Aug 2016</tspan>
        </text>
        <text x="496.6363636363636" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:165px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="412" opacity="1">
            <tspan>Sep 2016</tspan>
        </text>
        <text x="672.0909090909091" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:165px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="412" opacity="1">
            <tspan>Oct 2016</tspan>
        </text>
        <text x="847.5454545454546" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:165px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="412" opacity="1">
            <tspan>Nov 2016</tspan>
        </text>
        <text x="1023" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:165px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="412" opacity="1">
            <tspan>Dec 2016</tspan>
        </text>
        <text x="1198.4545454545455" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:165px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="412" opacity="1">
            <tspan>Jan 2017</tspan>
        </text>
        <text x="1373.909090909091" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:165px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="412" opacity="1">
            <tspan>Feb 2017</tspan>
        </text>
        <text x="1549.3636363636365" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:165px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="412" opacity="1">
            <tspan>Mar 2017</tspan>
        </text>
        <text x="1724.818181818182" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:165px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="412" opacity="1">
            <tspan>Apr 2017</tspan>
        </text>
        <text x="1900.2727272727275" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:165px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="412" opacity="1">
            <tspan>May 2017</tspan>
        </text>
    </g>
    <g class="highcharts-axis-labels highcharts-yaxis-labels ">
        <text x="43" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:649px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="398" opacity="1">
            <tspan>0</tspan>
        </text>
        <text x="43" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:649px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="334" opacity="1">
            <tspan>1,000</tspan>
        </text>
        <text x="43" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:649px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="270" opacity="1">
            <tspan>2,000</tspan>
        </text>
        <text x="43" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:649px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="206" opacity="1">
            <tspan>3,000</tspan>
        </text>
        <text x="43" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:649px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="142" opacity="1">
            <tspan>4,000</tspan>
        </text>
        <text x="43" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:649px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="78" opacity="1">
            <tspan>5,000</tspan>
        </text>
        <text x="43" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:649px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="14" opacity="1">
            <tspan>6,000</tspan>
        </text>
    </g>
    <rect x="570" y="10" width="78" height="383" fill="rgba(51,92,173,0.25)" class="highcharts-selection-marker"></rect>
    <g class="highcharts-label highcharts-tooltip highcharts-color-undefined" style="cursor:default;pointer-events:none;white-space:nowrap;" transform="translate(433,-9999)" opacity="0" visibility="visible">
        <path fill="rgba(255, 255, 255, 0)" class="highcharts-label-box highcharts-tooltip-box" d="M 0 0 L 58 0 64 -6 70 0 128 0 C 128 0 128 0 128 0 L 128 102 C 128 102 128 102 128 102 L 0 102 C 0 102 0 102 0 102 L 0 0 C 0 0 0 0 0 0"></path>
    </g>
</svg>