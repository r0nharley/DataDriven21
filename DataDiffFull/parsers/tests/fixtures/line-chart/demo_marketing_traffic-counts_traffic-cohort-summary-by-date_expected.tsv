Jul 2016	First Contacts	5133.0
Jul 2016	Shows	1565.0
Jul 2016	Apps Submitted	695.0
Jul 2016	Apps Approved	501.0
Jul 2016	Move-Ins	437.0
Aug 2016	First Contacts	5395.0
Aug 2016	Shows	1473.0
Aug 2016	Apps Submitted	660.0
Aug 2016	Apps Approved	477.0
Aug 2016	Move-Ins	401.0
Sep 2016	First Contacts	4544.0
Sep 2016	Shows	1260.0
Sep 2016	Apps Submitted	626.0
Sep 2016	Apps Approved	464.0
Sep 2016	Move-Ins	409.0
Oct 2016	First Contacts	4254.0
Oct 2016	Shows	1184.0
Oct 2016	Apps Submitted	581.0
Oct 2016	Apps Approved	431.0
Oct 2016	Move-Ins	378.0
Nov 2016	First Contacts	3382.0
Nov 2016	Shows	936.0
Nov 2016	Apps Submitted	453.0
Nov 2016	Apps Approved	326.0
Nov 2016	Move-Ins	281.0
Dec 2016	First Contacts	2268.0
Dec 2016	Shows	563.0
Dec 2016	Apps Submitted	257.0
Dec 2016	Apps Approved	196.0
Dec 2016	Move-Ins	178.0
Jan 2017	First Contacts	4267.0
Jan 2017	Shows	980.0
Jan 2017	Apps Submitted	410.0
Jan 2017	Apps Approved	317.0
Jan 2017	Move-Ins	288.0
Feb 2017	First Contacts	4812.0
Feb 2017	Shows	1121.0
Feb 2017	Apps Submitted	537.0
Feb 2017	Apps Approved	385.0
Feb 2017	Move-Ins	329.0
Mar 2017	First Contacts	5842.0
Mar 2017	Shows	1296.0
Mar 2017	Apps Submitted	655.0
Mar 2017	Apps Approved	471.0
Mar 2017	Move-Ins	329.0
Apr 2017	First Contacts	3086.0
Apr 2017	Shows	627.0
Apr 2017	Apps Submitted	278.0
Apr 2017	Apps Approved	159.0
Apr 2017	Move-Ins	54.0
May 2017	First Contacts	1.0
May 2017	Apps Submitted	1.0
May 2017	Apps Approved	1.0